applications/smart-distancing/ui/utils/visualization_utils.py

```diff
@@ -409,8 +409,8 @@ def visualization_preparation(nn_out, distances, dist_threshold):
     detection_boxes = []
```

Comment:
please remove this file changes from this PR, it is a fix for a different issue and already merged.